homeassistant/components/media_player/mpd.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, server, port, password, name):
 
         # set up MPD client
         self._client = mpd.MPDClient()
-        self._client.timeout = 5
+        self._client.timeout = 60
```

Comment:
do we need set this if they already are on default (60s)?

Reply:
This is default for MPD, not for python-mpd2, where default is None. I think we need timeout but more than 5 seconds.